charts/piraeus/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: piraeus
 description: A Helm chart for Piraeus Operator
 type: application
-version: 1.6.0
-appVersion: 1.6.0
+version: 1.7.0-rc.1
+appVersion: 1.16.0
```

Comment:
We usually only change these in the release commit.

Reply:
I plan to to a 1.7.0-rc.1tag, so that people can start testing, even if it's not 100% ready yet.

Reply:
I was actually just commenting on the fact that these are also touched in the "update container images" commit. GitHub doesn't show that very clearly. Wholly agree that we should update the versions for `1.7.0-rc.1`.

Reply:
Thanks for pointing that out. Removed that part of the automation. appVersion tracking LINSTOR version seems the most sensible to me, since the operator is already basically tied to the chart.

Reply:
I would stick with appVersion defining the operator version. It seems more logical to me since that is the "app" that the chart deploys. Also, we could do chart releases without operator releases, in which case we would only change version. In fact we even did this back in 0.2.1 :joy:

Reply:
Done. Now I still need to investigate why our tests say the container won't stop...

Reply:
Turns out it actually waited for the container to start again, and 30s was not enough for the test setup. increasing the timeout solved the issue. so nothing blocking this from my side.